### DISCUSSION TITLE
How to access gameSessionManifest(Looking to access game info like shot impacts, headshots, positioning, etc)

### DISCUSSION THREAD

**Discussion by Theoya** [saul/demofile-net#105] [Q&A] [answered]
Unsure if gameSessionManifest is even where I should be looking for this stuff, but could someone point out where i can access some of this information? (Shot impacts, headshots, positioning, etc)

**Comment by saul** [comment, marked as answer]
The game session manifest likely contains details for the Steam Game Coordinator about the lobby. I don't think you'll find anything useful about gameplay (impacts, headshots, positions etc) in there.

- Shot impacts: use the `demo.Source1GameEvent.BulletImpact`
- Headshots: use the `demo.Source1GameEvent.PlayerHurt` and `demo.Source1GameEvent.PlayerDeath` (see the `Hitgroup` property on the event)
- Positioning: this can be retrieved at any point from the player pawn: `e.AttackerPawn.Origin`

Hitgroup is a number that maps into this enum:

```c#
enum HitGroup
{
    Generic = 0,
    Head = 1,
    Chest = 2,
    Stomach = 3,
    Leftarm = 4,
    Rightarm = 5,
    Leftleg = 6,
    Rightleg = 7,
    Neck = 8,
    Unused = 9,
    Gear = 10,
    Special = 11,
};
```

Hope that helps!

**Reply by Theoya** [reply]
Awsome, thanks man ill try it out
